foundations/foundation-log/src/main/java/org/apache/servicecomb/foundation/log/core/element/LogItem.java

```diff
@@ -15,17 +15,22 @@
  * limitations under the License.
  */
 
-package org.apache.servicecomb.transport.rest.vertx.accesslog.element;
+package org.apache.servicecomb.foundation.log.core.element;
 
-import org.apache.servicecomb.transport.rest.vertx.accesslog.AccessLogParam;
+import org.apache.servicecomb.core.event.InvocationFinishEvent;
+import org.apache.servicecomb.core.event.ServerAccessLogEvent;
 
 /*
  * Access log item represents the items supported in access log printing.
  * It generate the segment of access log according to {@link AccessLogParam}
  */
-public interface AccessLogItem<T> {
+public interface LogItem<T> {
```

Comment:
Server access log and request log can both called AccessLog. Log is a general name may be confusing with any application trace. I'd prefer using AccessLogItem. And 'servicecomb.assesslog.request` as the configuration.

Reply:
ok